src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,35 @@
+package racingcar.model;
+
+public class Car {
+    public static final int MINIMUM_POSITION = 0;
+    public static final int MOVE_FORWARD = 4;
+    private final String name;
+    private int position;
```

Comment:
상수와 필드는 분리를 하는게 더 가독성 측면에서 좋은거 같습니다 :) 
```suggestion
    public static final int MINIMUM_POSITION = 0;
    public static final int MOVE_FORWARD = 4;
    
    private final String name;
    private int position;
```

Reply:
✅